literature.bib

```diff
@@ -564,55 +561,8 @@ @article{DGGM:POPL23
 	keywords = {String constraints, String solving, Word equations}
 }
 
-@article{AB:POPL23,
-	author = {Aguirre, Alejandro and Birkedal, Lars},
-	title = {{Step-Indexed Logical Relations for Countable Nondeterminism and Probabilistic Choice}},
-	year = 2023,
-	publisher = ACM,
-	address = NY,
-	volume = {7},
-	doi = {10.1145/3571195},
-	journal = PACMPL,
-	number = {POPL},
-	month = JAN,
-	articleno = {2},
-	numpages = {28},
-	keywords = {Functional Languages, Probabilistic programming, Logical Relations}
-}
-
-@article{CHH+:POPL23,
-	author = {Chappe, Nicolas and He, Paul and Henrio, Ludovic and Zakowski, Yannick and Zdancewic, Steve},
-	title = {{Choice Trees: Representing Nondeterministic, Recursive, and Impure Programs in Coq}},
-	year = 2023,
-	publisher = ACM,
-	address = NY,
-	volume = {7},
-	doi = {10.1145/3571254},
-	journal = PACMPL,
-	number = {POPL},
-	month = JAN,
-	articleno = {61},
-	numpages = {31},
-	keywords = {Interaction Trees, Concurrency, Nondeterminism, Formal Semantics}
-}
-
-@article{DGGM:POPL23,
-	author = {Day, Joel D. and Ganesh, Vijay and Grewal, Nathan and Manea, Florin},
-	title = {{On the Expressive Power of String Constraints}},
-	year = 2023,
-	publisher = ACM,
-	address = NY,
-	volume = {7},
-	doi = {10.1145/3571203},
-	journal = PACMPL,
-	number = {POPL},
-	month = JAN,
-	articleno = {10},
-	numpages = {31},
-	keywords = {String constraints, String solving, Word equations}
-}
-
```

Comment:
Why are these entries removed? It doesn't seem theyre added somewhere else. This will break my papers.

Reply:
Duplicate entries.